exercises/state-of-tic-tac-toe/canonical-data.json

```diff
@@ -335,7 +335,7 @@
         },
         {
           "uuid": "47c048e8-b404-4bcf-9e51-8acbb3253f3b",
-          "description": "Invalid board",
+          "description": "Invalid board: O started",
```

Comment:
If we go down that road, we can't know any board is valid unless there is only one X. 
I would argue that here, we know O played first because the description and the error say so.